docs/set_delegate.md

```diff
@@ -3,31 +3,44 @@ title: Delegation
 author: Simon Boissonneault-Robert
 ---
 
-# Examples demonstrating delegation for the various address types
+# Delegation and Baking/Staking
+
+## What is baking (staking) in Tezos?
+Staking is a concept of participation in forming blocks in any blockchain using the Proof-of-Stake consensus algorithm. In Tezos we call this term "Baking". Everyone who holds Tezos coins can produce, sign, and validate blocks to get rewards in proportion to their stake. 
```

Comment:
I think it requires a minimum of 8,000 XTZ Tezos to bake. Might be worth validating the amount and adding a precision about it in the doc